python/ml4ir/base/features/feature_config.py

```diff
@@ -259,8 +260,7 @@ def define_inputs(self) -> Dict[str, Input]:
         """
 
         def get_shape(feature_info: dict):
-            # TODO: Expand to other shapes
-            return (1,)
+            return feature_info.get("shape", (1,))
```

Comment:
@Ullimague Added support for specifying feature shape in the feature config. This is useful for list of entities, etc.